package.json

```diff
@@ -354,7 +354,7 @@
       }
     ],
     "configuration": {
-      "title": "Vim Configuration",
```

Comment:
I noticed in my settings extension list, we were the odd-man out here with this verbosity